contracts/Redemption.vy

```diff
@@ -189,24 +184,14 @@ def get_latest_price() -> uint256:
 @internal
 @view
 def _get_latest_price() -> uint256:
-    oracle_price: uint256 = convert(self._get_oracle_price(), uint256)
-    pool_price: uint256 = CURVE_POOL.price_oracle()
-    if pool_price < oracle_price:
-        return oracle_price
-    return pool_price
-
-
-@internal
-@view
-def _get_oracle_price() -> int256:
     round_id: uint80 = 0
     price: int256 = 0
     started_at: uint256 = 0
     updated_at: uint256 = 0
     answered_in_round: uint80 = 0
     (round_id, price, started_at, updated_at, answered_in_round) = PRICE_FEED.latestRoundData()
     assert updated_at + 3600 > block.timestamp, "price too old"
-    return price
+    return convert(price, uint256)
```

Comment:
For reference the price data is sourced from this aggregate contract
https://etherscan.io/address/0x3ebeaca272ce4f60e800f6c5ee678f50d2882fd4#code
That merges this 2 sources to create a YFI/ETH price

* YFI/USD: 0xA027702dbb89fbd58938e4324ac03B58d812b0E1
* ETH/USD: 0x5f4eC3Df9cbd43714FE2740f5E3616155c5b8419